### PR TITLE
Update CM-2

### PR DESCRIPTION
**Pull request by vlewisRH**
Tracked via:
https://issues.redhat.com/browse/CMP-260

### CODE REVIEW

File: openshift-container-platform-4/policies/CM-Configuration_Management/component.yaml [hunk 1]
```diff
@@ -32,9 +32,31 @@
...
-        tracked via:
+        Configuration management is enforced through Ansible playbooks,
+        Ignition configurations and Operators.
+
+        Ansible features a state-driven resource model that describes
+        the desired state of computer systems and services, allowing
+        reliable and repeatable IT infrastructure configuration. The
+        configuration is documented on the the system [Repository] site.
+
+        Ignition is the utility that is used by Red Hat Enterprise Linux CoreOS(RHCOS)
```
Comment (redhatrises) [inline openshift-container-platform-4/policies/CM-Configuration_Management/component.yaml]: Nitpick: add a space: `CoreOS(RHCOS)` should be `CoreOS (RHCOS)`

File: openshift-container-platform-4/policies/CM-Configuration_Management/component.yaml [hunk 2]
```diff
@@ -32,9 +32,31 @@
   implementation_status: planned
   narrative:
     - text: |
-        A complete control response is planned. Engineering progress can be
-        tracked via:
+        Configuration management is enforced through Ansible playbooks,
+        Ignition configurations and Operators.
```
Comment (redhatrises) [inline openshift-container-platform-4/policies/CM-Configuration_Management/component.yaml]: @vlewisRH This is a great write up! Well done. Question: it seems like Ansible is being deprecated completely in favor of operators. Should we still mention Ansible here?

File: openshift-container-platform-4/policies/CM-Configuration_Management/component.yaml [hunk 3]
```diff
@@ -32,9 +32,31 @@
...
+
+        Ansible features a state-driven resource model that describes
+        the desired state of computer systems and services, allowing
+        reliable and repeatable IT infrastructure configuration. The
+        configuration is documented on the the system [Repository] site.
+
+        Ignition is the utility that is used by Red Hat Enterprise Linux CoreOS(RHCOS)
+        to manipulate disks during initial configuration, confirming all new machines
+        meet the declared configuration.
```
Comment (redhatrises) [inline openshift-container-platform-4/policies/CM-Configuration_Management/component.yaml]: We should probably add this write up for Ignition and MCO to the CoreOS component no?

File: openshift-container-platform-4/policies/CM-Configuration_Management/component.yaml (hunk 2 above)
Comment (vlewisRH) [reply openshift-container-platform-4/policies/CM-Configuration_Management/component.yaml]: Thank you for your review, Ill make the recommended changes.

**Pull request merged**